client/src/components/Sle/Sidebar.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import "./Sidebar.css";
+
+export default function Sidebar(props) {
+  return (
+    <div className="Sidebar">
+        <p className="title">{props.title}</p>
+    </div>
+    
+  );
+}
```

Comment:
Let's end all files with a newline (see that red thing on line 11 - that's what it means). I know this seems minor but they'll ask you to do that in industry too.